### PR TITLE
Allow Pending PVC caused by WaitForFirstConsumer strategy

### PR DESCRIPTION
**Pull request by Michkov**
PVC which are used by CI are not used until CI is triggered.
PVCs are created but not Bound due toStorageClass configuration
WaitForFirstConsumer. Allow Pending PVC to be marked as Healthy instead
of simulating first customer by job.

### CODE REVIEW

File: hack/bootstrap-cluster.sh [hunk 1]
```diff
@@ -41,6 +41,32 @@ kubectl patch argocd/openshift-gitops -n openshift-gitops -p '{"spec": {"server"
...
+            if obj.status.phase == "Bound" then
+              hs.status = "Healthy"
+              hs.message = obj.status.phase
+              return hs
+            end
+          end
+        end
+        hs.status = "Progressing"
+        return hs
+' --type=merge
```
Comment (Michkov) [inline hack/bootstrap-cluster.sh:68]: @sbose78 This line should be applied on stage cluster before merging.

**Comment by mmorhun** [comment]
Could you update the description with instructions on how to test the MR?

**Comment by jduimovich** [comment]
To test this, you should simply be able to 
1. get a clusterbot cluster
2. run  bootstrap.sh 
3. all of the unbound PVC components will show as healthy instead of pending due to the patch in the script.

Note - this cannot be demonstrated on CRC as the PVCs are bound immediately on CRC. 

> @sbose78 This line should be applied on stage cluster before merging.
As the bootstrap is not run on staging this needs to be done manually by someone with access.
Alternatively we can run as a job in the gitops repo to ensure it gets configured this way like we do for the tektonconfig

**Comment by Michkov** [comment]
This is tested via ci/prow/appstudio-e2e-deployment  since it creates new cluster and deploy there applications, waiting for build and application-service to be synced -> if this would not work then it would never be synced and it would wait for Bound state of PVC. But test passed so it did not wait forever for bound of the PVC.

**Review by mmorhun — APPROVED** [review]

File: hack/bootstrap-cluster.sh (hunk 1 above)
Comment (Michkov) [reply hack/bootstrap-cluster.sh:68]: @jgwest Could you run this command on stage cluster?

**Pull request merged**